### PR TITLE
Added minimum and maximum player to game info

### LINKED ISSUE #4: As a gamer, I want to know how many can play a certain game, so that I don't invite too few people over 😭
- [ ] Add player count to the game description object
- [ ] Add some way for the frontend to fetch player count from the backend
- [ ] Add a filter so you can filter out games you are too few/many to play

### PR DESCRIPTION
**Pull request by EmilMadsstuen**
fixes #4

### CODE REVIEW

**Comment by EmilMadsstuen** [comment]
I think I fixed the API doc as requested. If not I'm not sure what to do :(

File: backend/src/routers/gameRouter.ts [hunk 1]
```diff
@@ -24,7 +24,9 @@ const gameRouter = Router();
  *    "description": "Game 1 description",
  * 	  "platformName": "Steam",
  *	  "releaseDate": "2023-04-13",
```
[the remark this answers is not in the document]
Comment (Supergamer1337) [reply backend/src/routers/gameRouter.ts]: We should probably update the documentation, but I would assume that this should be changed in the future.

**Review by Supergamer1337 — APPROVED** [review]
LGTM 👍

**Review by Oscariremma — APPROVED** [review]
Looks good 👍

**Pull request merged**